docs/api/README.md

```diff
@@ -123,12 +128,12 @@ The payload will look more like this standard JSON:
 
 ### Below is the detailed documentation for each field
 
-| FieldName     | Example       | Description     |
-| ------------- | ------------- | --------------- |
-| host_name | `"l3af-local-test"` | The host's name |
-| iface | `"fakeif0"` | Interface name |
-| bpf_programs | `""` | List of eBPF program names |
-| xdp_ingress | `""` | Names of xdp ingress type eBPF programs |
-| tc_ingress | `""` | Names of tc ingress type eBPF programs |
-| tc_egress | `""` | Names of tc egress type eBPF programs |
+| FieldName     | Example       | Description                           |
+| ------------- | ------------- |---------------------------------------|
+| host_name | `"l3af-local-test"` | The host's name                       |
+| iface | `"fakeif0"` | Interface name                        |
+| bpf_programs | `""` | List of BPF program names             |
+| xdp_ingress | `""` | Names of xdp ingress type BPF programs |
+| tc_ingress | `""` | Names of tc ingress type BPF programs |
+| tc_egress | `""` | Names of tc egress type BPF programs  |
```

Comment:
It was discussed in one of the TSC, its better to call BPF program than ebpf program, because now we are trying to discontinue loading from user program.